_includes/registration-button.html

```diff
@@ -1,7 +1,16 @@
 {% if page.registration_url %}
+{% if page.registration_url[0] %}
+<select class="btn btn--success registration-menu" >
+    <option value="" selected disabled hidden>&#xf56b; Register</option>
+    {% for uri in page.registration_url %}
+        <option value="{{ uri }}">Occurence #{{ forloop.index }}</option>
```

Comment:
```suggestion
        <option value="{{ uri }}">Occurrence #{{ forloop.index }}</option>
```

Reply:
sure, there is no functionality yet implemented, sorry, forgot to mention this :sweat_smile:

Reply:
I can do it this evening, but I already did want to show it to you to get your feedback about the appearance and the basic implementation